target/scripts/start-mailserver.sh

```diff
@@ -1138,6 +1138,62 @@ function _setup_ssl
 {
   _notify 'task' 'Setting up SSL'
 
+  # Primary certificate to serve for TLS
+  # NOTE: The `sed` substituion delimiter uses `~` instead of `/` due to file paths as values
+  function _set_certificate
+  {
+    local POSTFIX_KEY_WITH_FULLCHAIN=$1
+    local DOVECOT_KEY=$1
+    local DOVECOT_CERT=$1
+
+    # If 2nd param is provided, we've been provided separate key and cert instead of a fullkeychain
+    if [[ -n $2 ]]
+    then
+      local PRIVATE_KEY=$1
+      local CERT_CHAIN=$2
+
+      POSTFIX_KEY_WITH_FULLCHAIN="${PRIVATE_KEY} ${CERT_CHAIN}"
+      DOVECOT_KEY="${PRIVATE_KEY}"
+      DOVECOT_CERT="${CERT_CHAIN}"
+    fi
+
+    # Postfix configuration
+    # NOTE: `smtpd_tls_chain_files` expects private key defined before public cert chain
+    # May be a single PEM file or a sequence of files, so long as the order is key->leaf->chain
+    sed -i 's~^smtpd_tls_chain_files =.*~smtpd_tls_chain_files = '"${POSTFIX_KEY_WITH_FULLCHAIN}~" /etc/postfix/main.cf
+
+    # Dovecot configuration
+    sed -i 's~^ssl_key = <.*~ssl_key = <'"${DOVECOT_KEY}~" /etc/dovecot/conf.d/10-ssl.conf
```

Comment:
Noticed that this `sed` should actually be:

``` BASH
sed -i "s|^ssl_key = <.*|ssl_key = <${DOVECOT_KEY}|" /etc/dovecot/conf.d/10-ssl.conf
```

Which goes for the `sed` below too.

Reply:
I noticed you changed the `sed` delimiter from `~` to `|` is this a style convention for the project? They were previously `/` but that was changed by me due to the same character being used in filepath values. `_apply_tls_level` still kept the `/` delimiter for example.

Reply:
We are currently using `|` or `+` for `sed`, but honest to goodness, it's not that important actually:) I would be nice, but the `""` thing is more important. I figured, when you're at it, you can go ahead an change the delimiters too.

Reply:
I updated the `sed` substitution delimiters in the file (not just related to this PR), replacing any usage of `/` or `~`. After CI tests pass, should be good for merge.

Reply:
Oops, missed the merge window :sweat_smile: I'll add it as an additional PR then.